util/thread_local.cc

```diff
@@ -127,9 +127,6 @@ ThreadLocalPtr::StaticMeta* ThreadLocalPtr::Instance() {
   // is that thread_local supports dynamic construction and destruction of
   // non-primitive typed variables.  As a result, we can guarantee the
   // desturction order even when the main thread dies before any child threads.
-  // However, thread_local requires gcc 4.8 and is not supported in all the
-  // compilers that accepts -std=c++11 (e.g., the default clang on Mac), while
```

Comment:
@yiwu-arbug With XCode 8 which is now available on Mac, `thread_local` is now supported